scripts/utils.py

```
@@ -96,12 +94,11 @@
     return file_content
 
 
-def update_config_definition(base_url, selector, name, fileData, auth=None):
+def update_config_definition(base_url, selector, name, fileData, method="POST", auth=None, dry_run=False):
```

Pylint (3.3.7)

[refactor] Too many arguments (7/5)

(R0913)